esque/cli/options.py

```diff
@@ -19,15 +19,13 @@ def __init__(self):
             self.config = Config()
         except ConfigNotExistsException:
             click.echo(f"No config provided in {config_dir()}")
+            if not config_dir().exists():
```

Comment:
not an issue, but that method has a `exist_ok` param that prevents you from needing the `if`. Did you consider setting `parents=True` as well?

Reply:
Set `exist_ok=True`. Don't agree with the other one. :)